index.js

```diff
@@ -1,21 +1,29 @@
 /**
  * Picks objects keys by applying a predicate function.
  * @param  {Function} predicate The function applied for each key.
- * @return {Function} Returns the partially applied function.
+ * @return {Object} Returns the objects key.
  */
-function pickKeysBy (predicate) {
-  return function (originalObject) {
-    return Object.keys(originalObject)
-      .reduce((finalObject, key) => {
-        predicate(key) && (finalObject[key] = originalObject[key]);
-        return finalObject;
-      },
-    {});
-  };
+function pickKeysBy (originalObject, predicate) {
+  return Object.keys(originalObject)
+    .reduce((finalObject, key) => {
+      predicate(key) && (finalObject[key] = originalObject[key]);
+      return finalObject;
+    },
+  {});
 }
 
-// Check if a given key starts with `_`.
-const _isNotPrivate = key => key[0] !== '_';
-const omitPrivateKeys = pickKeysBy(_isNotPrivate);
+const omitPrivateKeys = function (originalObject, convention) {
```

Comment:
It may be nicer to partially apply the `convention`, so next calls wont need the param. Think about it as a "setup".